lib/Importer/ONNXModelLoader.cpp

```diff
@@ -4445,10 +4519,15 @@ Error ONNXModelLoader::loadOperator(const ONNX_NAMESPACE::NodeProto &op) {
 
   // Check if operator is supported in parent class, CommonOperatorLoader.
   bool tryLoadCommonOperatorResult;
-  ASSIGN_VALUE_OR_RETURN_ERR(tryLoadCommonOperatorResult,
-                             tryLoadCommonOperator(typeName, op, dict));
-  if (tryLoadCommonOperatorResult) {
-    return Error::success();
+  // The implementation of Softmax present in 'CommonOperatorLoader' handles
+  // only for opsetVersion < 13, So the new Softmax API is written in
+  // 'ONNXModelLoader' which will handle various opset versions.
+  if (typeName != "Softmax") {
```

Comment:
This will make us skip `Softmax` loading in CommonOperatorLoader for any ONNX opset though, right? Should we check opset here and fall back to CommonOperatorLoader? Or maybe call `tryLoadCommonOperator()` inside `ONNXModelLoader::loadSoftmax()` after we check the opset?

Reply:
When it sees the Softmax op, it does not go to CommonOperatorLoader and fall back at ONNXModelLoader, here we are handling the loading of Softmax according to its opset version, since the ONNX Softmax definition changes according to opset versions.
Or, should i check the opset version and perform only opset version 13 Softmax in ONNXModelLoader and fall back to CommonOperatorLoader for remaining? Let me know your thoughts